### PR TITLE
Dependency mappings for Python

### PR DESCRIPTION
**Pull request by sumanthkb44**
Dependency mappings for Python changes

### CODE REVIEW

File: src/LCT.PackageIdentifier/PythonProcessor.cs [hunk 1]
```diff
@@ -345,39 +356,51 @@ private static List<PythonPackage> ExecutePoetryCMD(string CommandForPoetry)
...
             else
             {
                 Logger.Debug($"ExecutePoetryCMD():Poetry CMD execution failed : " + result?.StdErr);
             }
 
+            return result;
+        }
+
+
+        private static List<PythonPackage> GetPackagesFromPoteryOutput(Result result)
```
Comment (karthika-g) [inline src/LCT.PackageIdentifier/PythonProcessor.cs:370]: correct the spelling in method name

File: src/LCT.PackageIdentifier/PythonProcessor.cs [hunk 2]
```diff
@@ -345,39 +356,51 @@ private static List<PythonPackage> ExecutePoetryCMD(string CommandForPoetry)
...
+        {
+            List<PythonPackage> packages = new List<PythonPackage>();
+            var strings = result.StdOut.Split(Environment.NewLine).ToList();
+
+            foreach (var package in strings)
+            {
+                var lst = package.Split(" ");
```
Comment (karthika-g) [inline src/LCT.PackageIdentifier/PythonProcessor.cs:377]: add comments if necessary

File: src/LCT.PackageIdentifier/PythonProcessor.cs [hunk 3]
```diff
@@ -405,13 +515,11 @@ public Task<ComponentIdentification> IdentificationOfInternalComponents(Componen
         public async Task<List<Component>> GetJfrogRepoDetailsOfAComponent(List<Component> componentsForBOM, CommonAppSettings appSettings, IJFrogService jFrogService, IBomHelper bomhelper)
         {
             // get the  component list from Jfrog for given repo
```
[the remark this answers is not in the document]
Comment (sumanthkb44) [reply src/LCT.PackageIdentifier/PythonProcessor.cs:517]: As of now JFrog doesn't store Python packages

File: src/LCT.PackageIdentifier/PythonProcessor.cs (hunk 2 above)
Comment (sumanthkb44) [reply src/LCT.PackageIdentifier/PythonProcessor.cs:377]: Added

File: src/LCT.PackageIdentifier/PythonProcessor.cs (hunk 1 above)
Comment (sumanthkb44) [reply src/LCT.PackageIdentifier/PythonProcessor.cs]: Corrected

File: src/LCT.PackageIdentifier/PythonProcessor.cs [hunk 4]
```diff
@@ -345,39 +356,51 @@ private static List<PythonPackage> ExecutePoetryCMD(string CommandForPoetry)
...
         {
+
             List<PythonPackage> lst = new List<PythonPackage>();
             string CommandForALlComp = "poetry show -C " + SourceFilePath;
             string CommandForMainComp = "poetry show --only main -C " + SourceFilePath;
+            string showCMD = "poetry show ";
 
-            List<PythonPackage> AllComps = ExecutePoetryCMD(CommandForALlComp);
-            List<PythonPackage> MainComps = ExecutePoetryCMD(CommandForMainComp);
+            List<PythonPackage> AllComps = GetPackagesFromPoteryOutput(ExecutePoetryCMD(CommandForALlComp));
```
[the remark this answers is not in the document]
Comment (sumanthkb44) [reply src/LCT.PackageIdentifier/PythonProcessor.cs]: Chnaged

**Review by karthika-g — APPROVED** [review]
looks good

**Pull request merged**